MicroProfile-Rest-Client/tck-runner/pom.xml

```diff
@@ -134,16 +134,80 @@
                 <microprofile.rest.client.api.version>1.1.payara-p2</microprofile.rest.client.api.version>
             </properties>
         </profile>
-        <!-- Payara 5 versions from 5.192 -->
+        <!-- Payara 5.192 and 193 - Rest Client 1.2 -->
+        <profile>
+            <id>payara5.192-193</id>
```

Comment:
Do we know if these runners are used by external parties? If they aren't, do we still need a profile for 192/193 as these versions have completed their cycles and not developed/worked on anymore? I could envisage the POMs getting quite messy as we implement more new versions and if it isn't necessary then why not start as we mean to go on - could still be required for something that I'm overlooking though

Reply:
Good point, 193 is the current release though, so I wouldn't be so quick at dismissing it. 

I was also playing with idea of creating tck runner project per major api version, but there are slight binary incompatibilities so e.g. 194 doesn't pass tck 1.1 anymore.

I'm all for removing anything older than 194 with release of 201.

And just verified 5.191.8 - still works ;).